testbed/tests/testbed.py

```diff
@@ -138,6 +148,12 @@ def run_tests(app, cov, args, report_coverage, run_slow):
     except ValueError:
         report_coverage = False
 
+    try:
+        args.remove("--ci")
+        running_in_ci = True
+    except ValueError:
+        running_in_ci = False
```

Comment:
It might be reasonable to replace this strategy with using `XFAIL` to mean the same thing; basically, all tests marked `XFAIL` would be required to pass in CI via the `--runxfail` flag. I haven't reviewed all the current uses of `XFAIL` but several of them, at least, conflict with such repurposing.

Reply:
I'm not sure this will work; we're currently using "skip" to mean "this feature hasn't been implemented yet" and "xfail" to mean either "this feature *can't* be implemented because the platform doesn't support it", or "testing this feature is unreliable on this platform in a way we're willing to ignore" (e.g., MapView testing on macOS x86).